docs/eslint/eslint-plugin-query.md

```diff
@@ -93,8 +93,8 @@ Alternatively, add `@tanstack/query` to the plugins section, and configure the r
 
 ## Rules
 
-- [@tanstack/query/exhaustive-deps](./exhaustive-deps)
-- [@tanstack/query/no-rest-destructuring](./no-rest-destructuring)
-- [@tanstack/query/stable-query-client](./stable-query-client)
-- [@tanstack/query/no-unstable-deps](./no-unstable-deps)
-- [@tanstack/query/infinite-query-property-order](./infinite-query-property-order)
+- [@tanstack/query/exhaustive-deps](../eslint/exhaustive-deps)
+- [@tanstack/query/no-rest-destructuring](../eslint/no-rest-destructuring)
+- [@tanstack/query/stable-query-client](../eslint/stable-query-client)
+- [@tanstack/query/no-unstable-deps](../eslint/no-unstable-deps)
+- [@tanstack/query/infinite-query-property-order](../eslint/infinite-query-property-order)
```

Comment:
I just found out the correct way to link relative links is with `./page.md` - the  `.md` extension is important!